ansible/README.md

```diff
@@ -2,33 +2,85 @@
 
 We use [Ansible](http://docs.ansible.com/) to manage our deployment.
 
+* [Setup](#setup)
+* [Hosts](#hosts)
+* [New Deployment](#new-deployment)
+* [Changing Secrets](#changing-secrets)
+* [Updating Configs](#updating-configs)
+* [Deploying](#deploying)
+
 ## Setup
 
 You just need a few things installed locally.
 
-#### Install Ansible with homebrew:
+### Install Ansible with Homebrew
 
 ```
 brew install ansible
 ```
 
+### Install third-party roles
+
 You then need to install any third party roles that we need via [ansible-galaxy](https://galaxy.ansible.com/docs/).
 Run the below from the `ansible` directory:
 
 ```
 ansible-galaxy install requirements.yml
 ```
 
+### Obtain the Ansible vault password file
+
+The Ansible vault password files are used to decrypt all Ansible vault files in this directory. Obtain the password file
+from 1Password, and place it in a convenient location on your local machine, like `~/.vault_password`. You will need
+this password file for most Ansible operations described below.
+
 ## Hosts
 
 These are the environments
```

Comment:
This might be a good place to describe what the environments are used for, perhaps? Staging vs demo, QA vs dev, for example.

Reply:
Shall we link [this document](https://docs.google.com/spreadsheets/d/1JCfFYetk9Jrtc5iUHp-7Fx5V3QqpuCWojjcEibRJN7I/edit#gid=0)? We already link this in the top level readme.

Reply:
I'd like to retire that spreadsheet in favor of: https://docs.google.com/document/d/1VTVBr8HcLWK6Nrg4gQkuQKb3H8EtiqQA-zGWTu3ddHc/edit#heading=h.gh0xebw6sceg

What do you think?

Reply:
Sure 👍🏼 Should we add a note to keep the list updated somewhere in either case? Wondering if we should pull that in the repo instead.

Reply:
Ooh great call. Add a note to the "new deployment" instructions to say "go add it to this doc please".